app/views/time/now.html.erb

```diff
@@ -0,0 +1 @@
+The current time is <%= Time.zone.now.strftime("%a, %e %b %Y %H:%M:%S") %> UTC +00:00
```

Comment:
You can do it all as one:  `<%= Time.zone.now.strftime('%a, %d %b %Y %H:%M:%S UTC %:z') %>`